File: math/fibonacci/python/fibonacci.py
def fib(n):  # return Fibonacci series up to n from: https://docs.python.org/2/tutorial/controlflow.html
    """Return a list containing the Fibonacci series up to n."""
    result = []
    a, b = 0, 1
    while a < n:
        result.append(a)
        a, b = b, a+b
    return result

def fibonacci(n):
	fib = [0, 1]
	
	for i in range(n+1):
		fib.append(fib[-1] + fib[-2])

	return fib[n]


for i in range(0,10):
	print(fibonacci(i))

File: math/fibonacci/go/fibonacci.go
package fibonacci

func Fib(n int32) int32 {
	var fibonacci [100]int32
	fibonacci[0], fibonacci[1] = 1, 1

	i := 2

	for ; i <= n; i++ {
		fibonacci[i] = fibonacci[i-1] + fibonacci[i-2]
	}

	return fibonacci[i-1]
}
